### PR TITLE
Neculaclaudiu/tema8

### PR DESCRIPTION
**Pull request by Claudiu357**

### CODE REVIEW

File: src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package ioananadroflorea.tests.Tema7;
 
-import ioanandroflorea.driver.BrowserManager;
+import ioanaandroflorea.driver.BrowserManager;
```
Comment (95teo) [inline src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java:3]: Please don't modify this file. Revert this change. Thanks!

File: src/test/java/neculaclaudiu/tests/seleniumpart3/tests/AllTests.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        page.selectCategory("Desktop");
+        page.selectLanguage("Visual Basic");
+        page.clickOnCodeInButton();
+        BasicAjaxVerificationPage verificationPage = new BasicAjaxVerificationPage(driver);
+        Assert.assertEquals(verificationPage.getCategoryText(),"2");
+        Assert.assertEquals(verificationPage.getLanguageText(),"13");
+    }
+
+    @Test(groups = {"Desktop", "Mobile"})
+    public void JavaScriptRedirectsPageTest(){
```
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/seleniumpart3/tests/AllTests.java:22]: camelCase when naming methods as per Java conventions. Thanks!

File: src/test/java/neculaclaudiu/tests/seleniumpart3/tests/AllTests.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    public void refreshPageTest(){
+        RefreshPage page = new RefreshPage(driver);
+        String refresh1 = page.getRefreshNumber();
+        driver.navigate().refresh();
+        String refresh2 = page.getRefreshNumber();
+        Assert.assertNotEquals(refresh1,refresh2);
+    }
+
+    @Test(groups = {"Desktop", "Mobile"})
+    public void SimpleDynamicButtonsPageTest(){
```
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/seleniumpart3/tests/AllTests.java:43]: camelCase when naming methods as per Java conventions. Thanks!

File: src/test/java/neculaclaudiu/tests/seleniumpart3/tests/AllTests.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        SimpleDynamicButtonsPage page = new SimpleDynamicButtonsPage(driver);
+        page.clickOnStartButton();
+        page.clickOnOneButton();
+        page.clickOnTwoButton();
+        page.clickOnThreeButton();
+        Assert.assertEquals(page.buttonMessageGetText(), "All Buttons Clicked");
+    }
+
+    @Test(groups = {"Desktop", "Mobile"})
+    public void DisabledDynamicButtonsTest(){
```
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/seleniumpart3/tests/AllTests.java:53]: camelCase when naming methods as per Java conventions. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]
Please don't forget to ask for a colleague review too before being able to do the merge. Thanks!

**Review by sorinfoca — APPROVED** [review]

**Pull request merged**